packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -624,21 +630,34 @@ export async function getFaceMatchState(
 }
 
 export async function isFaceDistanceBelowThreshold({
+  sessionInformation,
   faceDetector,
   videoEl,
   ovalDetails,
   reduceThreshold = false,
-  isMobile = false,
 }: {
+  sessionInformation: SessionInformation;
   faceDetector: FaceDetection;
   videoEl: HTMLVideoElement;
   ovalDetails: LivenessOvalDetails;
   reduceThreshold?: boolean;
-  isMobile?: boolean;
 }): Promise<{
   isDistanceBelowThreshold: boolean;
   error?: ErrorState;
 }> {
+  const challengeConfig =
+    sessionInformation?.Challenge?.FaceMovementAndLightChallenge
+      ?.ChallengeConfig;
+  if (
+    !challengeConfig ||
+    !challengeConfig.FaceDistanceThreshold ||
+    !challengeConfig.FaceDistanceThresholdMin
+  ) {
+    throw new Error('Challenge config not returned from session information.');
+  }
```

Comment:
Can we handle this validation before calling `isFaceDistanceBelowThreshold`? Ideally we would handle this validation where the challenge is returned before updating the state machine with the `challengeConfg`